appveyor.yml

```diff
@@ -6,7 +6,7 @@ build_script:
 
 # to run your custom scripts instead of automatic tests
 test_script:
-  - gradlew.bat checkstyleMain checkstyleTest pmdMain jacocoTestReport jacocoRootReport test --stacktrace -Pheadless=true -PlogTests
+  - gradlew.bat checkstyleMain checkstyleTest pmdMain findbugsMain jacocoTestReport jacocoRootReport test --stacktrace -Pheadless=true -PlogTests
```

Comment:
Does this run on the tests too? We want to find possible bugs in the tests too right?